### ISSUE TITLE
feature request: Sync to folder on game exit

### ISSUE THREAD

**Issue by hartmark** [crschnick/pdx_unlimiter#109]
I love this program as it's the easiest way of getting my steam deck and main rig save games in sync.

The only thing missing to get it even better is to have an option to get the sync to a folder trigger automatically. Preferably on game exit or fixed intervals.

I have the folder set up to be synced  with my cloud at filen.io so I'm certain to have the latest saves at all times.

**Comment by hartmark** [comment]
never mind, I might have misunderstood the "Sync/Backup" storage function as it is only one way sync to get all saves stored safely in a folder and not to be used to import on another machine.

![image](https://github.com/crschnick/pdx_unlimiter/assets/12826053/06d506f9-4e07-40f5-851f-dd344480abc7)

If I have this folder synced with my cloud I get what I expect instead. But I think an export/import function would be more clear and less error prone.

**Comment by crschnick** [comment]
Yeah maybe I should clarify the sync functionality name and add a tip to the tooltip for the storage directory that it can be shared via a cloud provider.

It is not called export because it only selectively exports save games that are not present yet in the target directory and it does not preserve all information within pdxu. So if you were to reimport all of these save games, you would lose some information like custom names, notes, etc.

I guess I can add a setting to suppress the addons window.

I just need to find some time for that, same issue with the AUR package that I wasn't able to update yet.

**Comment by hartmark** [comment]
I'd like a way to easily be able to share the state with notes and everything using cloud.

I noticed that the program will do a write of the files when exiting the program so I need to make sure to quit it before syncing on the other machine so I get consistent state otherwise I can lose some games.

How's it going with the AUR package? Is there anything I can help with?

**Comment by crschnick** [comment]
Do you play on multiple systems in parallel, i.e. with multiple pdxu instances open? Otherwise sharing the storage directory via the cloud should work fine and sync everything.

Building the AUR package should be pretty easy, but I'm very short on time right now.

**Comment by hartmark** [comment]
I was playing on my regular pc and when getting back on the steam deck the pdxu was already running and didn't detect that the files had changed on disk do I tried to quit pdxu but it just wrote out the state it has already in memory and lost the new incoming saves
